### PR TITLE
I18n: Final Czech translation 🇨🇿

### PR DESCRIPTION
**Pull request by Aditoo17**
Last one before 2.8.0.

### CODE REVIEW

File: config/locales/cs.yml [hunk 1]
```diff
@@ -880,9 +883,11 @@ cs:
     featured_tags: Zvýrazněné hashtagy
     identity_proofs: Důkazy identity
     import: Import
+    import_and_export: Import a export
     migrate: Přesunutí účtu
     notifications: Oznámení
     preferences: Předvolby
+    profile: Profil
```
Comment (ykzts) [inline config/locales/cs.yml]: This line is unnecessary.

File: config/locales/cs.yml [hunk 2]
```diff
@@ -880,9 +883,11 @@ cs:
     featured_tags: Zvýrazněné hashtagy
     identity_proofs: Důkazy identity
     import: Import
+    import_and_export: Import a export
```
Comment (ykzts) [inline config/locales/cs.yml]: This line is unnecessary.

File: config/locales/cs.yml [hunk 3]
```diff
@@ -871,6 +872,8 @@ cs:
     revoke_success: Relace úspěšně zamítnuta
     title: Relace
   settings:
+    account: Účet
+    account_settings: Nastavení účtu
```
Comment (ykzts) [inline config/locales/cs.yml]: This line is unnecessary.

File: config/locales/cs.yml [hunk 4]
```diff
@@ -871,6 +872,8 @@ cs:
     revoke_success: Relace úspěšně zamítnuta
     title: Relace
   settings:
+    account: Účet
```
Comment (ykzts) [inline config/locales/cs.yml]: This line is unnecessary.

**Review by ykzts — CHANGES_REQUESTED** [review]

**Comment by Aditoo17** [comment]
@ykzts I don't get it. Those lines were added in #10533. I'm not sure what's going on.

**Comment by ykzts** [comment]
Ah, understood. Your branch is a bit old. You need to merge the master branch.

**Review by ykzts — APPROVED** [review]

**Pull request merged**